simulations/vip-460/zksyncmainnet.ts

```diff
@@ -0,0 +1,145 @@
+import { expect } from "chai";
+import { BigNumber } from "ethers";
+import { formatUnits, parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { setMaxStalePeriodInChainlinkOracle } from "src/utils";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+import ERC20_ABI from "src/vip-framework/abi/erc20.json";
+import { checkIsolatedPoolsComptrollers } from "src/vip-framework/checks/checkIsolatedPoolsComptrollers";
+import { checkRiskParameters } from "src/vip-framework/checks/checkRiskParameters";
+import { checkVToken } from "src/vip-framework/checks/checkVToken";
+import { checkInterestRate } from "src/vip-framework/checks/interestRateModel";
+
+import vip460, {
+  COMPTROLLER,
+  ZKETH_ORACLE,
+  convertAmountToVTokens,
+  newMarket,
+  tokens,
+} from "../../vips/vip-460/bscmainnet";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+import ZKETH_ORACLE_ABI from "./abi/ZkETHOracle.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const BLOCKS_PER_YEAR = BigNumber.from("31536000");
+
+const { POOL_REGISTRY, NORMAL_TIMELOCK, CHAINLINK_ORACLE, RESILIENT_ORACLE } = NETWORK_ADDRESSES["zksyncmainnet"];
+
+const WETH_CHAINLINK_FEED = "0x6D41d1dc818112880b40e26BD6FD347E41008eDA";
+
+forking(56240200, async () => {
+  const provider = ethers.provider;
+  const oracle = new ethers.Contract(RESILIENT_ORACLE, RESILIENT_ORACLE_ABI, provider);
+  const poolRegistry = new ethers.Contract(POOL_REGISTRY, POOL_REGISTRY_ABI, provider);
+  const comptroller = new ethers.Contract(COMPTROLLER, COMPTROLLER_ABI, provider);
+
+  before(async () => {
+    await setMaxStalePeriodInChainlinkOracle(
+      CHAINLINK_ORACLE,
+      tokens["WETH"].address,
+      WETH_CHAINLINK_FEED,
+      NORMAL_TIMELOCK,
+    );
+  });
+
+  describe("vTokens deployment", () => {
+    checkVToken(newMarket.vToken.address, newMarket.vToken);
+  });
+
+  testForkedNetworkVipCommands("zksync-zkETH", await vip460());
+
+  describe("Post-VIP state", () => {
+    describe("Oracle configuration", async () => {
+      it("has the correct WETH price", async () => {
+        const price = await oracle.getPrice(tokens["WETH"].address);
+        expect(price).to.be.eq(parseUnits("2715.45552182", 18));
```

Comment:
This test failed on my machine. The expected value was `2740.42477014`

Reply:
Have you set fork block number to 56240200 for anvil-zksync fork?

```
anvil-zksync --gas-price 2000000000 fork --fork-url mainnet --fork-block-number 56240200
```